src/components/Editor.tsx

```diff
@@ -0,0 +1,52 @@
+import { useEffect, useRef } from "react";
+import EmailEditor, { UnlayerOptions, User } from "react-email-editor";
+import mergeTags from "../external/merge.tags";
+
+export interface EditorProps {
+  design?: any;
+}
+
+export const Editor = (props: EditorProps) => {
+  const { design } = props;
```

Comment:
What do you think about:

```suggestion
export const Editor = ({ design }: EditorProps) => {
```

Reply:
I plan to reuse the editor component and delegate the specific functionality to parent components (campaing | template).